project/invitro/exports.py

```diff
@@ -1,9 +1,28 @@
+from copy import copy
 from django.db.models.loading import get_model
 
 from utils.helper import FlatFileExporter
 
 
-class IVEndpointFlatDataPivot(FlatFileExporter):
```

Comment:
I think it's ok; see line 25 below. I just renamed the class and pulled-out some shared helper functions